Formula/libplist.rb

```diff
@@ -21,29 +21,41 @@ class Libplist < Formula
   end
 
   option "with-python", "Enable Cython Python bindings"
+  option "with-python3", "Enable Cython Python3 bindings"
 
   depends_on "pkg-config" => :build
   depends_on "libxml2"
   depends_on :python => :optional
+  depends_on :python3 => :optional
 
   resource "cython" do
-    url "http://cython.org/release/Cython-0.21.tar.gz"
-    sha256 "0cd5787fb3f1eaf8326b21bdfcb90aabd3eca7c214c5b7b503fbb82da97bbaa0"
+    url "https://github.com/cython/cython/archive/0.24.tar.gz"
+    sha256 "b60b91f1ec88921a423d5f0a5e2a7c232cdff12d9130088014bf89d542ce137b"
   end
 
   def install
-    ENV.deparallelize
     args = %W[
       --disable-dependency-tracking
       --disable-silent-rules
       --prefix=#{prefix}
     ]
 
-    if build.with? "python"
+    if build.with? "python3"
+      version = Language::Python.major_minor_version "python3"
       resource("cython").stage do
-        ENV.prepend_create_path "PYTHONPATH", buildpath+"lib/python2.7/site-packages"
-        system "python", "setup.py", "build", "install", "--prefix=#{buildpath}",
-                 "--single-version-externally-managed", "--record=installed.txt"
+        ENV.prepend_create_path "PYTHONPATH", buildpath+"lib/python#{version}/site-packages"
```

Comment:
Yeah, there was a mini discussion about it after the `audit` issue where the `pkgshare` audit was only bothering to check `/` rather than `+` and none of us noticed because most of our path creation is done using `/` now. So it makes sense to stop mixing & matching. `+` has a valid use case, but simple path creation should be adopting `/` ideally.